src/components/CreateResolution/CompleteResolutionSummary.vue

```diff
@@ -155,6 +168,22 @@ export default class CompleteResolutionSummary extends Mixins(DateMixin) {
     const result = validationItemsDetails.length === invalidValidationItems.length
     return result
   }
+
+  private async reRenderResolutionText (): Promise<void> {
+    this.resolutionTextHeightUpdateRequired = false
+    this.$refs.resolutionTextRef.calculateInputHeight()
+  }
+
+  // Previously, the resolution text area would not render to the appropriate height relative to the amount of content
+  // when navigating from another step.  In hooking into the visibility change event on the resolution text area via the
+  // v-observe-visibility property, we are able to force a re-calculation of the text area height when a user navigates
+  // to the complete resolution summary step from another step for the first time. This results in the text area being
+  // rendered to the appropriate height.
+  private async onResolutionVisibilityChanged (isVisible, entry) {
```

Comment:
Also not async :)

I'm curious to see this in action. Looks neat.

Reply:
removed `async`.  yeah, it's kinda cool...this directive.  I could see it coming in handy for other stuff.

Reply:
Could you do a small tech talk on it?

Reply:
sure, i'll talk about it a bit at the BCROS developer demo thing.